tests/float/test_float2.models.expected

```diff
@@ -1,5 +1,7 @@
 
 unknown
+; This model is a best-effort. It includes symbols
+; for which model generation is known to be incomplete.
```

Comment:
For this one, it seems the test is running without enabling the `fpa` theory:
```
  (rule
   (target test_float2.models_tableaux.output)
   (deps (:input test_float2.models.smt2))
   (package alt-ergo)
   (action
    (chdir %{workspace_root}
     (with-stdout-to %{target}
      (ignore-stderr
       (with-accepted-exit-codes 0
        (run %{bin:alt-ergo}
             --timelimit=2
             --enable-assertions
             --output=smtlib2
             --frontend dolmen
             --sat-solver Tableaux
             %{input})))))))
```

Reply:
My point is that it is fine to not print anything since replying "unknown" is already saying that the model may be incomplete. At least we should remove this when #917 is taken care of.